components/user_settings/modal/user_settings_modal.jsx

```diff
@@ -76,7 +76,7 @@ class UserSettingsModal extends React.Component {
         this.state = {
             active_tab: 'general',
             active_section: '',
-            prev_active_section: '',
+            prev_active_section: 'dummySectionName',
```

Comment:
Can you elaborate on the thinking behind `dummySectionName` vs. checking for just the empty string?

Reply:
I don't know why but it's the default section title for each section https://github.com/mattermost/mattermost-webapp/blob/b0839434d1ef02c4268ed5d4d4e4f72c2b37eed5/components/user_settings/general/user_settings_general.jsx#L82

Reply:
Ah, that "makes sense". (To be honest, this whole block of code is a little confusing, but that's not your fault!)

Can you add that comment to make this explicit?
```suggestion
            prev_active_section: 'dummySectionName', // dummy value that should never match any section name
```